org.eclipse.xtext.tests/testdata/wizard-expectations/org.xtext.example.lsMavenApp/org.xtext.example.lsMavenApp.parent/pom.xml

```diff
@@ -16,6 +16,17 @@
 		<module>org.xtext.example.lsMavenApp</module>
 		<module>org.xtext.example.lsMavenApp.ide</module>
 	</modules>
+	<dependencyManagement>
```

Comment:
seeing how this has developed and the maven-plugin test failures i am no longer sure this is a good idea